cli/src/actions/path.ts

```diff
@@ -43,6 +43,11 @@ export async function promptPath(ctx: Context) {
         initialValue: false,
       });
 
+      if (typeof confirmation === "symbol") {
```

Comment:
ah right i should've caught this! i was even looking at it at first being like "this can be a symbol, the boolean check should handle it"